src/node_contextify.h

```diff
@@ -9,6 +9,8 @@
 namespace node {
 namespace contextify {
 
+enum SourceType { kScript = 6633, kModule };
```

Comment:
why the number?

Reply:
to stay out of the way of other users of v8's API, fwiw it's "node" on a phone